gravitee-apim-console-webui/src/portal/customization/api/api-list/portal-api-list.component.scss

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2021 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use 'sass:map';
+@use '../../../../../node_modules/@angular/material/index' as mat;
+@use '../../../../../node_modules/@gravitee/ui-particles-angular/index' as gio;
+
+$typography: map.get(gio.$mat-theme, typography);
+
+.menu-api-container {
+  display: flex;
+  flex-direction: column;
+  gap: 48px;
+}
+
+.menu-api {
+  &__content {
+    display: flex;
+    flex-flow: column;
+    gap: 48px;
+
+    &__header {
+      display: flex;
+      flex-direction: column;
+      gap: 0;
+      padding-bottom: 16px;
+
+      &__title {
+        display: flex;
+        flex-flow: row;
+        gap: 8px;
+
+        & > h3 {
+          margin: 0;
+        }
+      }
+
+      &__description {
+        @include mat.m2-typography-level($typography, body-2);
+      }
+    }
+  }
+}
+
+.menu-api__content__header__title-row {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.mat-formfield {
```

Comment:
Avoid using classes specific to material design. It can cause problems during upgrades when Material changes class names. You can use the parent classes to control its width.

Reply:
Done